packages/studio-plugin/src/ParsingOrchestrator.ts

```diff
@@ -131,6 +133,11 @@ export default class ParsingOrchestrator {
       if (fileExists) {
         this.pageNameToPageFile[pageName] = this.getPageFile(pageName);
       }
+    } else if (filepath.startsWith(this.paths.layouts)) {
```

Comment:
do we need to consider removing deleted layouts from the `layoutNames` here?

Reply:
I thought about that-- I don't think users will have the ability to delete layouts.  Is that the situation that you're talking about, or is there another reason why we would delete layouts from `layoutNames`?

Also, I think that array just a placeholder for now, and we'll do more in depth parsing in later items.

Reply:
I was thinking about that-- I don't think users will have the ability to delete layouts.  Is that the situation that you're talking about, or is there another reason why we would delete layouts from `layoutNames`?

Also, I think that array just a placeholder for now, and we'll do more in depth parsing in later items.

Reply:
I was thinking about HMR, where if someone (presumably a developer) deletes a layout file in their repo while running Studio, then we'd want to delete the layout from `layoutNames`, similar to how we do for components (which also can't be deleted through the UI but could be by a developer).

I figured since it should be pretty quick to add, it would be better to add it now, even though the array is just a placeholder, so we don't forget it once we add the full parsing. Especially, since we already have some logic here (unlike for `hotReloadStore`), I could see us changing the existing logic in a future PR and forgetting to consider the deletion case